openedx.yaml

```diff
@@ -5,4 +5,6 @@ owner: edx/platform-team
 nick: edx-enterprise
```

Comment:
Nit: The owner above doesn't seem correct?

Reply:
it's so wrong.